README.md

```diff
@@ -1,29 +1,43 @@
-# warehouse-models
-Data models for the [Warehouse][warehouse.ai]. Built on top of Cassandra and Datastar.
+# `warehouse-models`
 
-## API Documentation
+[![Version npm](https://img.shields.io/npm/v/warehouse-models.svg?style=flat-square)](https://www.npmjs.com/package/warehouse-models)
+[![License](https://img.shields.io/npm/l/warehouse-models.svg?style=flat-square)](https://github.com/warehouseai/warehouse-models/blob/master/LICENSE)
+[![npm Downloads](https://img.shields.io/npm/dm/warehouse-models.svg?style=flat-square)](https://npmcharts.com/compare/warehouse-models?minimal=true)
+[![Build Status](https://travis-ci.org/warehouseai/warehouse-models.svg?branch=master)](https://travis-ci.org/warehouseai/warehouse-models)
+[![Dependencies](https://img.shields.io/david/warehouseai/warehouse-models.svg?style=flat-square)](https://github.com/warehouseai/warehouse-models/blob/master/package.json)
 
-All of the objects returned from this module have the same api as [`datastar`](https://github.com/godaddy/datastar) with the schemas as mentioned later.
+Data models for the [Warehouse][warehouse.ai]. Built on top of [Cassandra] and
```

Comment:
Nit, but is warehouse referred to as `the Warehouse` anywhere else? 
![Drop the "the"](https://media.giphy.com/media/OtKMqumpq0oj6/giphy.gif)